shared/config.lua

```diff
@@ -43,6 +43,10 @@ Config.NearestShopBlipUpdateDelay = 10000
 Config.InvincibleDuringCustomization = true
 
 Config.PreventTrackerRemoval = true -- Disables "Scarf and Chains" section if the player has tracker
+Config.TrackerClothingOptions = {
+    item = 13,

```

Comment:
Can you please rename this to drawable? I would like this to be more consistent, atleast in the config. And I have plans to change the item to drawable in other places as well so it will be a 1 less thing for me to modify

Reply:
Sure, I have a question. Will something like data["accessory"].item be renamed to drawable and moved to the compatibility layer, or will it remain the same?

Reply:
The outfit data format compatibility  will stay the same for some time. I'm just trying to make sure that anything new that is added, doesn't use the old conventions with `item` as the key and follows the new data format